packages/test-runner/src/cli.ts

```diff
@@ -101,7 +101,7 @@ async function runStage1(command) {
     }
   });
   runner = new Runner(config, new Multiplexer(reporterObjects));
-  if (!runner.loadFiles(testDir, filteredArguments.slice(1), command.testMatch, command.testIgnore).success)
```

Comment:
I put the success property there so I would know whether true or false was good or bad.

Reply:
Let's keep it simple - returning true for success is wide spread enough to imply it.

Reply:
Reverted